README.adoc

```diff
@@ -99,13 +99,7 @@ Setup and usage is described in our *https://resilience4j.readme.io/docs[User Gu
 
 == Overview
 
-Resilience4j provides several core modules and add-on modules:
-
-=== All core modules and the Decorators class
-
-* resilience4j-all
```

Comment:
I think we should keep resilience4j-all on the readme

Reply:
It is mentioned below as "TIP: For core modules package or `Decorators`(...)"